charts/newrelic-prometheus-agent/tests/curated_experience_test.yaml

```diff
@@ -0,0 +1,147 @@
+suite: test configmap with CuratedExperience
+templates:
+  - templates/configmap.yaml
+tests:
+  - it: config with CuratedExperience true
+    set:
+      licenseKey: license-key-test
+      cluster: cluster-test
+      config:
+        kubernetes:
+          curated_experience:
+            enabled: true
+        # Set empty to make this test simple
+        static_targets:
+    asserts:
+      - equal:
+          path: data.config\.yaml
+          value: |-
+            # Configuration for newrelic-prometheus-configurator
+            common:
+              external_labels:
+                cluster_name: cluster-test
+              scrape_interval: 30s
+            kubernetes:
+              jobs:
+              - job_name_prefix: kubernetes-job
+                target_discovery:
+                  endpoints: true
+                  filter:
+                    annotations:
+                      prometheus.io/scrape: true
+                  pod: true
+              curated_experience:
+                regexes:
```

Comment:
naming looks differnet, in the values is additional_regexes

Reply:
yes, I do not like it, but currently, it is expected, 
```
regexes = additional_regexes+curated experience one
```